src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -151,6 +151,5 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     }
   }
 
-  eP->present("");
```

Comment:
To REALLY fix it, I would need to replace all LM_F for LM_Ts ...
Do we want this here?